### PR TITLE
Feat/input

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
Screenshot from 2022-02-10 02-04-14
Screenshot from 2022-02-10 02-04-26

### CODE REVIEW

File: component/input/input.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.form-group {
+  margin: 0.5rem 0.5rem;
```
Comment (abhijeetekad) [inline component/input/input.css]: No need to right twice if all sides have the same value.

**Review by abhijeetekad — APPROVED** [review]
Everything looks fine. Just do suggested changes and you can merge.

File: component/input/input.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+label {
+  display: block;
+  font-weight: 400;
+}
+.form-group input {
+  padding: 0.25rem 0.25rem;
```
Comment (AmishMishra11) [inline component/input/input.css]: use shorthand property

**Review by AmishMishra11 — COMMENTED** [review]
looks good just do the requested changes

**Pull request merged**